docs/migrating.md

```diff
@@ -1 +1,84 @@
+| `SIGNALFX_HTTP_SERVER_ERROR_STATUSES`                  | Comma-separated list of HTTP server response statuses for which the spans are set as errors, for example: `300, 400-599`.                                                                                                     | `500-599`                           | none                                                     |
+| `SIGNALFX_INSTRUMENTATION_ELASTICSEARCH_TAG_QUERIES`   | Enable the tagging of an Elasticsearch command PostData as `db.statement`. It might introduce overhead for direct streaming users.                                                                                            | `true`                              | none                                                     |
+| `SIGNALFX_INSTRUMENTATION_MONGODB_TAG_COMMANDS`        | Enable the tagging of a Mongo command BsonDocument as `db.statement`.                                                                                                                                                         | `true`                              | Not configurable using environment variable.             |
+| `SIGNALFX_INSTRUMENTATION_REDIS_TAG_COMMANDS`          | Enable the tagging of a Redis command as `db.statement`.                                                                                                                                                                      | `true`                              | Not configurable using environment variable.             |
+| `SIGNALFX_LOGS_INJECTION`                              | Enable to inject trace IDs, span IDs, service name, and environment into logs.                                                                                                                                                | `false`                             | Logs correlated if `Microsoft.Extensions.Logging` used.  |
+| `SIGNALFX_TRACE_DELAY_WCF_INSTRUMENTATION_ENABLED`     | Enable the updated WCF instrumentation that delays execution until later in the WCF pipeline when the WCF server exception handling is established.                                                                           | `false`                             | none                                                     |
+| `SIGNALFX_TRACE_HEADER_TAGS`                           | Comma-separated map of HTTP header keys to tag names, automatically applied as tags on traces.                                                                                                                                | `"x-my-header:my-tag,header2:tag2"` | none                                                     |
```

Comment:
What's about OTEL_RESOURCE_ATTRIBUTES?

Reply:
I don't think `OTEL_RESOURCE_ATTRIBUTES` should be mentioned here.
Value for attributes added with `OTEL_RESOURCE_ATTRIBUTES` are static, with `SIGNALFX_TRACE_HEADER_TAGS` value for a tag is extracted from a header.
`OTEL_RESOURCE_ATTRIBUTE` is already mentioned as a equivalent for `SIGNALFX_GLOBAL_TAGS`.